contracts/mock/math/DSMathMock.sol

```diff
@@ -0,0 +1,15 @@
+// SPDX-License-Identifier: ALGPL-3.0-or-later-or-later
+// from https://github.com/makerdao/dss/blob/master/src/jug.sol
+pragma solidity ^0.8.9;
+
+import {DSMath} from "../../math/DSMath.sol";
+
+contract DSMathMock {
+    function rpow(
+        uint256 x,
+        uint256 n,
+        uint256 b
+    ) public pure returns (uint256 z) {
```

Comment:
Why `public`? Can be `external`. `z` is not used, can be removed (`uint256` should stay)